### ISSUE TITLE
The evaluation tool.

### ISSUE THREAD

**Issue by GongXinyuu** [cs-chan/Exclusively-Dark-Image-Dataset#1]
Hi, thanks for your great work!

I'm wondering that is there any evaluation tool for your dataset?

**Comment by cs-chan** [comment]
Thank you for your interest in our work.

We used the MatConvNet MATLAB toolbox by VLFeat for all the image classification experiments with CNN. You can get the toolbox, pre-trained models, and demo scripts on how to get started at this link: http://www.vlfeat.org/matconvnet/

As we have described in Section IV of our paper (https://arxiv.org/abs/1805.11227), we specifically used the imagenet-resnet-50-dag model and fine-tuned it using different ratios of bright and low-light images from the training set to obtain multiple models. The models are then used to classify the same testing set where we then compare their classification accuracy. The training, validation, and testing set annotations are available in our github page: https://github.com/cs-chan/Exclusively-Dark-Image-Dataset/tree/master/Groundtruth
